lib/notify.js

```diff
@@ -28,6 +28,7 @@ module.exports = CoreObject.extend({
     var opts        = opts || {};
     var plugin      = this._plugin;
     var makeRequest = Promise.denodeify(this._client);
+    var critical = (('critical' in opts) ? delete opts.critical : false);
```

Comment:
i wanted to delete the option to prevent passing it to the httpClient.
As you can use any client that option could break the request.
I believe i better to extract it from the options but if you prefer i can change it like you said.